### PR TITLE
Use custom leaderboard title if present

### LINKED ISSUE #2035: Changing leaderboard title in settings does not change title on leaderboard page
selection_002

The sidebar item name and the tab name on the course settings page are both changed, but not the page title on the leaderboard itself.

### PR DESCRIPTION
**Pull request by jeremyyap**
Fixes #2035

### CODE REVIEW

File: app/views/course/leaderboards/show.html.slim [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= page_header
+= page_header(@settings.title)
```
Comment (allenwq) [inline app/views/course/leaderboards/show.html.slim]: I think you need to check for `nil`, use announcement index page as a reference ?

Comment (jeremyyap) [inline app/views/course/leaderboards/show.html.slim]: The helper function uses the default translation if title is nil 

https://github.com/Coursemology/coursemology2/blob/master/app/helpers/application_helper.rb#L35

Comment (allenwq) [inline app/views/course/leaderboards/show.html.slim]: Correct, I forgot about that.  But need to use `format_inline_text(@settings.title)` to remove dangerous tags.

Comment (jeremyyap) [inline app/views/course/leaderboards/show.html.slim]: Okay, changed

**Pull request merged**